Tests/InstallationTests/CommonFiles/RevenueCat-Swift.h

```diff
@@ -1,4 +1,4 @@
-// Generated by Apple Swift version 5.10 (swiftlang-5.10.0.13 clang-1500.3.9.4)
+// Generated by Apple Swift version 6.0 effective-5.10 (swiftlang-6.0.0.9.10 clang-1600.0.26.2)
```

Comment:
From what I've checked the changes in this file should be ok, but I wanted to double check with iOS people to make sure. cc @RevenueCat/sdk